pytripgui/plan_executor/executor.py

```diff
@@ -0,0 +1,45 @@
+import os
+import logging
+
+from pytripgui.model.config_model import Trip98ConfigModel
+import pytrip.tripexecuter as pte
+from pytripgui.plan_executor.simulation_results import SimulationResults
+
+logger = logging.getLogger(__name__)
+
+
+class PlanExecutor:
+    def __init__(self):
+        self.trip_config = Trip98ConfigModel()
+
+    def check_config(self):
+        # TODO replace with
```

Comment:
replace with what ?

Reply:
done:
# TODO replace with function that actually runs trip, and collect returned errors